### ISSUE TITLE
Copy of commit message not updated whrn force-pushing a PR

### ISSUE THREAD

**Issue by sschuberth** [rtyley/submitgit#21]
The copy of the commit message as shown in the submitGit web interface is not updated when e.g. fixing a typo in the commit message and force-pushing the PR. For example, https://submitgit.herokuapp.com/git/git/pull/163 still shows the `"--quite"` typo at the very beginning of the commit message, whereas clicking on the SHA1 1cb92e18 in the `Id` column shows that the typo has been fixed to say `"--quiet"` in the commit.

**Comment by rtyley** [comment]
The message seen on submitGit is actually a bit of half-finished functionality, which I should probably turn off until it's actually finished. It's not actually your commit message - it's your **pull request description**, which GitHub initially _defaults_ to your commit message, if there is only one commit. After that, if you force-push a different commit message, the pull request description will not change, tho' obviously the GitHub web interface will let you edit it.

![image](https://cloud.githubusercontent.com/assets/52038/8846092/4debca0e-311c-11e5-8a9e-d5e9ded5530d.png)

The idea is to optionally make the pull request description into the _cover letter_ for the patch bomb (which would cover #9) - but that was before I ended up launching submitGit a little early, and I haven't got round to that feature yet.

**Comment by sschuberth** [comment]
I see. So suppose I would edit the PR's description in the GitHub web interface, would that change also be reflected in submitGit's web interface? If not, is that a planned feature?

**Comment by rtyley** [comment]
> So suppose I would edit the PR's description in the GitHub web interface, would that change also be reflected in submitGit's web interface?

Yup, it should do that right now, possibly with a 1-second delay due to caching.

**Comment by rtyley** [comment]
Tho right now, the displayed field doesn't actually _do_ anything, obviously.

**Comment by sschuberth** [comment]
So this issue is more or less invalid, thanks for the clarification. But could we maybe make more clear in submitGit's web interface that the shown text is not directly linked to a commit's message? Maybe add a header above that text field saying something like "Cover letter text" and maybe below that, in smaller letters,  "This text is derived from the PR's description" or something like that?
